packages/oidc-client/src/renewTokens.ts

```diff
@@ -34,7 +34,8 @@ export async function renewTokensAndStartTimerAsync(oidc, refreshToken, forceRef
     if(configuration.storage === window.sessionStorage && !serviceWorker) {
         tokens = await syncTokens(oidc, refreshToken, forceRefresh, extras);
     } else {
-        tokens = await navigator.locks.request(lockResourcesName, async (lock) => {
+        tokens = await navigator.locks.request(lockResourcesName, { ifAvailable: true }, async (lock) => {
```

Comment:
I think it will require some little ajustement. But let published like it as an alpha. Thank you @abhilashlr !

Reply:
Sure @guillaume-chervet ! Let me know what needs to be changed, I can work on it :)